wlm_server/pid/dac/dac8734.py

```diff
@@ -0,0 +1,78 @@
+"""Module for DAC8734 controlled via an ArtyS7 FPGA."""
+
+import serial
+from qcodes_driver.AD9912.ArtyS7 import ArtyS7
+
+from .base import BaseDAC
+
+class DAC8734(BaseDAC):
+    """DAC8734 controlled via an ArtyS7 FPGA.
+    
+    Constants:
+        NUM_CHANNELS: Number of channels on the DAC8734.
+        VREF: Reference voltage for voltage-to-code conversion in V.
+        MIN_V: Minimum allowed output voltage in V.
+        MAX_V: Maximum allowed output voltage in V.
+    """
+
+    NUM_CHANNELS = 8
+    VREF = 2.5
+    MIN_V, MAX_V = 0, 2.5
+
+    def __init__(self, port: str):
+        """Extended."""
+        super().__init__(port)
+        self._fpga: ArtyS7 | None = None
+
+    def open(self):
+        """Overridden."""
+        if self._is_open:
+            print(f'DAC8734 on {self._port} is already open')
+            return
+        try:
+            self._fpga = ArtyS7(self._port)
+        except serial.SerialException as e:
+            print(f'Failed to open DAC8734: {e}')
+        else:
+            self._is_open = True
```

Comment:
How about calling `super().open()`?
If you think the current way is better, how about removing the implementation of `BaseDAC.open()`, so leaving it as an empty abstract method?